### DISCUSSION TITLE
How to build an MI model capable of predicting 2022 world cup results

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#140] [Talks in English]
**Speaker: Luis Vinicius**

Join to learn, how to build a model capable of predicting possible results and winner of the 2022 World Cup using python and its libraries focused on data science, and the extraction of data from FIFA.

First, the lecture will be presented how the extraction of FIFA data with python was performed and what data was used for the construction of the Machine Learning model. After that, it will be discussed how the development of the machine learning model was carried out and the choice of algorithm to perform the prediction of possible results, it will also be talked about a little how the possible results are calculated through the algorithm and finally present what are the results that the model was able to predict. In the end, if there is time left, it will be presented how the deployment of the model and application construction was performed for the model to be available on the web and be used by other people who want to know the possible results that the model can predict.

Application: https://possiveis-resultados-qatar2022.herokuapp.com/

**Comment by shyanukant** [comment]
### 5 Steps of Machine Learning or MLops
- Data Analysis
- Feature Engineering
- Feature Selection
- Model Building
- Model Deployment
